server/src/main/java/org/opensearch/cluster/service/ClusterApplier.java

```diff
@@ -49,6 +49,12 @@ public interface ClusterApplier {
      */
     void setInitialState(ClusterState initialState);
 
+    /**
+     * Sets the committed state for the applier.
+     * @param clusterState state that has been committed by cluster-manager
+     */
+    void setCommitState(ClusterState clusterState);
```

Comment:
why do we need this method at all? ClusterService.state() is the applied state already